src/containers/course-section/resource-item/ResourceItem.tsx

```diff
@@ -93,9 +87,11 @@ const ResourceItem: FC<ResourceItemProps> = ({
     const type = resourceType ?? resource.resourceType
     return resourceIcons[type] ?? null
   }, [resourceType, resource.resourceType])
-
-  const status = availability?.status ?? ResourceAvailabilityStatusEnum.Open
-
+  const status = Object.values(ResourceAvailabilityStatusEnum).includes(
+    availability?.status as ResourceAvailabilityStatusEnum
+  )
+    ? (availability?.status as ResourceAvailabilityStatusEnum)
+    : ResourceAvailabilityStatusEnum.Closed
```

Comment:
Why did you changed that?

Reply:
Because there was statement which manually set status on Open. I replace that statement with changing to current status or if we are having null, we'll get Closed status